src/steamship/agents/examples/simple_character_without_tools.py

```diff
@@ -0,0 +1,49 @@
+from typing import Type
+
+from pydantic import Field
+
+from steamship.agents.llms.openai import ChatOpenAI
+from steamship.agents.service.agent_service import AgentService
+from steamship.agents.simple.agent_without_reasoning import AgentWithoutReasoning
+from steamship.invocable import Config
+from steamship.utils.repl import AgentREPL
+
+DEFAULT_NAME = "Harry Potter"
+DEFAULT_TAGLINE = "famous wizard, the one who lived, defeater of Voldemort"
+DEFAULT_PERSONALITY = """You chat with your fans about your adventures in the wizarding world. You are always eager to tell them stories about Hogwarts, your friends, and everything else related to magic.
+
+Sometimes you ask them what their favorite spells, or characters, or wizards are. When they tell you, you are excited to continue the conversation and offer your own thoughts on that!"""
+
+
+class SimpleCharacterWithoutTools(AgentService):
```

Comment:
This is totally fine. But, it opens the question of: what is the transition story?

If I make a Harry Potter agent without Tools and love it, but one day wish it could also draw pictures of castles, how do I update my Harry Potter agent to now have a Tool?